nrt/data/IEEE14/IEEE14_SyntaxExamples/IEEE14_IterationAndTimeSteps/reference/outputs2/curves/curves.csv

```diff
@@ -0,0 +1,38 @@
+time;NETWORK__BUS____1_TN_Upu_value;NETWORK__BUS____2_TN_Upu_value;NETWORK__BUS____3_TN_Upu_value;NETWORK__BUS____4_TN_Upu_value;NETWORK__BUS____5_TN_Upu_value;NETWORK__BUS____6_TN_Upu_value;NETWORK__BUS____7_TN_Upu_value;NETWORK__BUS____8_TN_Upu_value;NETWORK__BUS____9_TN_Upu_value;NETWORK__BUS___10_TN_Upu_value;NETWORK__BUS___11_TN_Upu_value;NETWORK__BUS___12_TN_Upu_value;NETWORK__BUS___13_TN_Upu_value;NETWORK__BUS___14_TN_Upu_value;GEN____1_SM_generator_omegaPu;GEN____1_SM_generator_PGen;GEN____1_SM_generator_QGen;GEN____1_SM_generator_UStatorPu;GEN____1_SM_voltageRegulator_EfdPu;GEN____2_SM_generator_omegaPu;GEN____2_SM_generator_PGen;GEN____2_SM_generator_QGen;GEN____2_SM_generator_UStatorPu;GEN____2_SM_voltageRegulator_EfdPu;GEN____3_SM_generator_omegaPu;GEN____3_SM_generator_PGen;GEN____3_SM_generator_QGen;GEN____3_SM_generator_UStatorPu;GEN____3_SM_voltageRegulator_EfdPu;GEN____6_SM_generator_omegaPu;GEN____6_SM_generator_PGen;GEN____6_SM_generator_QGen;GEN____6_SM_generator_UStatorPu;GEN____6_SM_voltageRegulator_EfdPu;GEN____8_SM_generator_omegaPu;GEN____8_SM_generator_PGen;GEN____8_SM_generator_QGen;GEN____8_SM_generator_UStatorPu;GEN____8_SM_voltageRegulator_EfdPu;_LOAD___2_EC_load_PPu;_LOAD___2_EC_load_QPu;
+0.000000;1.059970;1.045041;1.010001;1.017703;1.019553;1.070149;1.061530;1.089926;1.055982;1.051051;1.057013;1.055333;1.050525;1.035657;1.000000;232.329584;-16.691112;1.058824;1.436651;1.000000;40.126172;43.675435;1.048778;1.269364;1.000000;-0.074832;25.065946;1.011505;1.112250;1.000000;0.005609;12.802790;1.070149;1.461749;1.000000;-0.006407;17.569940;1.096374;1.194231;0.216990;0.126990;
```

Comment:
There is not 0.4s between the lines?

Reply:
The dumps defined in the task are not alone in the crv file. That's why we don't see the 0.4s minimum offset between each dumps.

Reply:
not sure I got your answer, the only reason I see to dump a line before the 0.4s would be an event, and that would be a lot of event?

Reply:
In the Simulation::simulate() method, there is a lot of updateCurves calls besides the call with the timeStep. That's why there are that many dumps.